_posts/2016-08-01-tips-for-freshmen-by-nuswhispers.markdown

```diff
@@ -0,0 +1,144 @@
+---
+layout: post
+title:  "Tips For Freshmen By NUSWhispers"
+date:   2016-07-31 17:20:00
+author: Nicholette Li
+author_thumbnail: 610496971
+---
+
+Due to the unfortunate suspension of Freshman Orientation Week camps, kind souls from NUSWhispers have emerged to share their tips and experiences with incoming freshmen who missed their opportunity to seek advice from their seniors. The admins from NUSWhispers have decided to collate them in a single post for easier reading.
```

Comment:
suggestion: add a link to the facebook post / nus whisper posts that collates all these information?